tests_metricflow/snapshots/test_offset_metrics.py/str/DuckDB/test_offset_to_grain_with_grain_smaller_than_offset_non_default__query_output.txt

```diff
@@ -0,0 +1,260 @@
+test_name: test_offset_to_grain_with_grain_smaller_than_offset_non_default
```

Comment:
No issues related to this bug in this output - just added this to confirm that nothing changed here for `offset_to_grain` queries